src/auxiliary/p4est.jl

```diff
@@ -116,9 +116,25 @@ coarsen_p4est!(p8est::Ptr{p8est_t}, recursive, coarsen_fn_c, init_fn_c) = p8est_
 
 # Create new ghost layer from p4est, only connections via faces are relevant
 # 2D
-new_ghost_layer_p4est(p4est::Ptr{p4est_t}) = p4est_ghost_new(p4est, P4est.P4EST_CONNECT_FACE)
+new_ghost_p4est(p4est::Ptr{p4est_t}) = p4est_ghost_new(p4est, P4est.P4EST_CONNECT_FACE)
 # 3D
-new_ghost_layer_p4est(p8est::Ptr{p8est_t}) = p8est_ghost_new(p8est, P4est.P8EST_CONNECT_FACE)
+new_ghost_p4est(p8est::Ptr{p8est_t}) = p8est_ghost_new(p8est, P4est.P8EST_CONNECT_FACE)
+
+# Check if ghost layer is valid
+# 2D
+function ghost_is_valid_p4est(p4est::Ptr{p4est_t}, ghost_layer::Ptr{p4est_ghost_t})
+  return p4est_ghost_is_valid(p4est, ghost_layer)
+end
+# 3D
+function ghost_is_valid_p4est(p4est::Ptr{p8est_t}, ghost_layer::Ptr{p8est_ghost_t})
+  return p8est_ghost_is_valid(p4est, ghost_layer)
+end
+
+# Destroy ghost layer
+# 2D
+ghost_destroy_p4est(ghost_layer::Ptr{p4est_ghost_t}) = p4est_ghost_destroy(ghost_layer)
```

Comment:
Minor question for Mr. Consistency: Should we also use `ghost_new_p4est` for consistency?

Reply:
Yes, you are right. In case it has been done (mostly) consistently thus far (i.e., move the `p4est` from being a prefix to being a suffix), it should also be done this way here.

Reply:
Renamed it d2954bf.